src/logger.js

```diff
@@ -3,12 +3,17 @@
 import { Logger, type LoggerType } from "@krakenjs/beaver-logger/src";
 import { memoize } from "@krakenjs/belter/src";
 
+import { getDisableSetCookie } from "./global";
 import { getPayPalLoggerUrl } from "./domains";
 
 type GetLogger = () => LoggerType;
 
+const loggerUrl = getDisableSetCookie()
+  ? `${getPayPalLoggerUrl()}?disable-set-cookie=true`
```

Comment:
Originally I thought this would work but now I'm second guessing it. Can we pass query params like this on the logger HTTP Post request? 🤔

Reply:
We can pass it! I was able to test this out. 😄 As long as Stargate can receive this (which they should be able to), we should be good: 
![image](https://github.com/paypal/paypal-sdk-client/assets/60204385/9a392d55-760f-46e5-b48d-8b1844613947)

Reply:
Great to hear! Thanks for testing this locally and ensuring it works. I'm cool with this solution 💯